cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesImport.java

```diff
@@ -48,14 +49,17 @@ public CgmesImport(PlatformConfig platformConfig, List<CgmesImportPostProcessor>
         this.defaultValueConfig = new ParameterDefaultValueConfig(platformConfig);
         this.postProcessors = Objects.requireNonNull(postProcessors).stream()
                 .collect(Collectors.toMap(CgmesImportPostProcessor::getName, e -> e));
+        Path boundary = platformConfig.getOptionalConfigDir()
+                .map(dir -> dir.resolve(FORMAT).resolve("boundary"))
+                .orElseThrow(() -> new PowsyblException("Cannot resolve boundary files loaction as configuration directory is not defined in platform config"));
```

Comment:
Do we want to always throw an exception? In many case we can just embed boundary files with the case and so we don't care about this directory.

Reply:
I agree, you can simply put `null` here and in `boundary(Properties p)` method:
```java
     String loc = ConversionParameters.readStringParameter(
                        getFormat(),
                        p,
                        boundaryLocationParameter,
                        defaultValueConfig);
        if (loc == null) {
             return null;
        }
        // Check that the Data Source has valid CGMES names
        ReadOnlyDataSource ds = new GenericReadOnlyDataSource(Paths.get(loc));
        if ((new CgmesOnDataSource(ds)).names().isEmpty()) {
            return null;
        }
        return ds;
```